build/yamllint.conf

```diff
@@ -0,0 +1,8 @@
+extends: default
```

Comment:
I don't suppose you can make this file the same name as the build pipeline? It's pretty self explanatory but that might make things easier to parse for someone who isn't familiar.